frontend/src/views/SettingsView.vue

```diff
@@ -43,38 +43,26 @@
       </div>
     </div>
     <!-- content -->
-    <div class="w-4/5 pt-14">
+    <div class="w-full pt-2 lg:w-4/5 lg:pt-14">
 
       <!-- general settings -->
       <settings-general v-if="activeView === settingsSections.general" />
 
       <!-- calendar settings -->
       <settings-calendar v-if="activeView === settingsSections.calendar" />
 
-      <!-- appointments and booking settings -->
-      <div v-if="activeView === settingsSections.appointmentsAndBooking">
-        <div class="text-3xl font-semibold text-gray-500">{{ t('heading.appointmentsAndBookingSettings') }}</div>
```

Comment:
thought: Should we delete those lang snippets in the language files as well? To keep things clean?